ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/daemon-terminattr.j2

```diff
@@ -4,27 +4,26 @@
 !
 daemon TerminAttr
 {%     set cvp_config.cli = "exec /usr/bin/TerminAttr" %}
-{%     for cluster_name in daemon_terminattr.clusters | arista.avd.natural_sort %}
-{%         set cluster = daemon_terminattr.clusters[cluster_name] %}
-{%         set cvp_config.cli = cvp_config.cli ~  " -cvopt " ~ cluster_name ~ ".addr=" ~ cluster.cvaddrs | join(',') %}
-{%         if cluster.cvauth.method is arista.avd.defined('key') %}
-{%             set cvp_config.cli = cvp_config.cli ~ " -cvopt " ~ cluster_name ~ ".auth=key," ~ cluster.cvauth.key | arista.avd.default('') %}
-{%         elif cluster.cvauth.method is arista.avd.defined('token') and cluster.cvauth.token_file is arista.avd.defined %}
-{%             set cvp_config.cli = cvp_config.cli ~ " -cvopt " ~ cluster_name ~ ".auth=token," ~ cluster.cvauth.token_file %}
-{%         elif cluster.cvauth.method is arista.avd.defined('token-secure') and cluster.cvauth.token_file is arista.avd.defined %}
-{%             set cvp_config.cli = cvp_config.cli ~ " -cvopt " ~ cluster_name ~ ".auth=token-secure," ~ cluster.cvauth.token_file %}
+{%     for cluster_name in daemon_terminattr.clusters | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
```

Comment:
I was asking to use use ```cluster``` instead of cluster_name because cluster_name doesn't sound apt (cluster_name is now cluster.name). Same in other template.

Reply:
corrected